scripts/migrate_operations_to_workflows.py

```
@@ -0,0 +1,277 @@
+                    verification_errors += 1
+                else:
+                    target_item = response['Item']
+                    # Compare key fields
+                    if (target_item.get('userId') != item.get('userId') or
+                        target_item.get('operationType') != item.get('operationType') or
+                        target_item.get('status') != item.get('status')):
+                        logger.error(f"Data mismatch for item: {operation_id}")
+                        verification_errors += 1
+                        
+            except ClientError as e:
+                logger.error(f"Error verifying item {operation_id}: {e}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)